examples/official-storybook/components/addon-measure/ShadowRoot.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { drawSelectedElement } from '@storybook/addon-measure/dist/cjs/box-model/visualizer';
+import { drawSelectedElement } from '@storybook/addon-measure/dist/esm/box-model/visualizer';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { init, destroy } from '@storybook/addon-measure/dist/cjs/box-model/canvas';
+import { init, destroy } from '@storybook/addon-measure/dist/esm/box-model/canvas';
```

Comment:
These imports will likely break over time

Reply:
Yeah, I am not sure what's going on here exactly (cc @winkerVSbecks) but I just figure this is a strict improvement.

Reply:
To test the addon I call an internal method from within stories. Instead of importing from the source, I ended up importing from the built addon. Probably a mistake. I can submit a PR to make this import relative.